data/data-pipeline/data_pipeline/score/score_m.py

```diff
@@ -189,21 +190,29 @@ def _climate_factor(self) -> bool:
             <= self.MEDIAN_HOUSE_VALUE_THRESHOLD
         )
 
-        self.df[field_names.EXPECTED_POPULATION_LOSS_RATE_LOW_INCOME_FIELD] = (
+        self.df[
+            field_names.EXPECTED_POPULATION_LOSS_RATE_LOW_INCOME_LOW_HIGHER_ED_FIELD
+        ] = (
             expected_population_loss_threshold
             & self.df[field_names.FPL_200_AND_COLLEGE_ATTENDANCE_SERIES]
         )
 
-        self.df[field_names.EXPECTED_AGRICULTURE_LOSS_RATE_LOW_INCOME_FIELD] = (
+        self.df[
+            field_names.EXPECTED_AGRICULTURE_LOSS_RATE_LOW_INCOME_LOW_HIGHER_ED_FIELD
+        ] = (
             expected_agriculture_loss_threshold
             & self.df[field_names.FPL_200_AND_COLLEGE_ATTENDANCE_SERIES]
         )
 
-        self.df[field_names.EXPECTED_BUILDING_LOSS_RATE_LOW_INCOME_FIELD] = (
+        self.df[
+            field_names.EXPECTED_BUILDING_LOSS_RATE_LOW_INCOME_LOW_HIGHER_ED_FIELD
+        ] = (
             expected_building_loss_threshold
             & self.df[field_names.FPL_200_AND_COLLEGE_ATTENDANCE_SERIES]
         )
 
+        # QQ: Should this be removed?
+        # note to self: leave as is for now
         self.df[
```

Comment:
Oh yeah, this should be removed. Want to go ahead and clean it up?

Reply:
sure